index.html

```diff
@@ -143,33 +143,56 @@ <h1 class="work-title">Tonic</h1>
                 <li>
                     <div class="languages">
                         <p>Languages</p>
-                        <img src="./assets//images/arrow-down.svg" alt="" srcset="">
+                        <img src="./assets/images/arrow-down.svg" alt="arrow-down" srcset="">
                     </div>
                     <ul class="languages-list">
                         <li>
-                            <img src="./assets/images/js.svg" alt="" srcset="">
+                            <img src="./assets/images/js.svg" alt="javascript" srcset="">
                             <p>Javascript</p>
                         </li>
                         <li>
-                            <img src="./assets/images/html.svg" alt="" srcset="">
+                            <img src="./assets/images/html.svg" alt="html" srcset="">
                             <p>HTML</p>
                         </li>
                         <li>
-                            <img src="./assets/images/css.svg" alt="" srcset="">
+                            <img src="./assets/images/css.svg" alt="css" srcset="">
                             <p>CSS</p>
                         </li>
                     </ul>
                 </li>
                 <li class="flex-list">
                    <p>Frameworks</p>
-                   <img src="./assets/images/right-arrow.svg" alt="" srcset="">
+                   <img src="./assets/images/right-arrow.svg" alt="right-arrow" srcset="">
                 </li>
                 <li class="flex-list">
                    <p>Skills</p>
-                   <img src="./assets/images/right-arrow.svg" alt="" srcset="">
+                   <img src="./assets/images/right-arrow.svg" alt="right-arrow" srcset="">
                 </li>
             </ul>
         </section>
+        <section class="contact">
+            <div class="contact-me-info">
+                <h1>Contact me</h1>
+                <p class="contact-me-info-text">
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+I                        ’d love to help with it!
+                </p>
+            </div>
+            <form action="https://formspree.io/f/xwkjzklr" method="post" class="contact-form">
+                <ul>
+                    <li class="form-control">
+                        <input required placeholder="Your name" name="name" id="name" type="text" class="form-input"/>
+                    </li>
+                    <li class="form-control">
+                        <input required placeholder="Your email" name="email" id="email" type="email" class="form-input"/>
+                    </li>
+                    <li class="form-control">
+                        <textarea required placeholder="Write your message here"  name="message" id="message" cols="20" rows="10" class="form-input"/></textarea>
+                    </li>
```

Comment:
- [ ] For this section, `Name` input should have `maxlength="30"` attribute.
- [ ] Message input has `maxlength="500"` attribute.
- These are necessary validations to keep your code professional

## Requirement :
![image](https://github.com/habitoz/portofolio/assets/50990925/332608f6-8540-4493-8005-b381b01f26ea)